src/Compilers/Shared/DesktopBuildClient.cs

```diff
@@ -473,13 +480,13 @@ private static bool CheckPipeConnectionOwnership(NamedPipeClientStream pipeStrea
             (ObjectSecurity)typeof(PipeStream)
             .GetTypeInfo()
             .GetDeclaredMethod("GetAccessControl")
```

Comment:
I think it's OK if this throws. The only usage is surrounded in a try-catch that swallows all exceptions and turns it into a failed build response.